test/ctmrg/gradients.jl

```diff
@@ -36,8 +31,8 @@ gradmodes = [
         LinSolver(; solver=KrylovKit.GMRES(; tol=gradtol), iterscheme=:fixed),
         LinSolver(; solver=KrylovKit.GMRES(; tol=gradtol), iterscheme=:diffgauge),
     ],
-    [
-        nothing,
+    [  # Only use :diffgauge due to high gauge-sensitivity (perhaps due to small χenv?)
```

Comment:
did you re check this after the gaugefixing fixes?

Reply:
Okay I checked again and the gauge-fixing fixes don't make a difference to the high gauge-sensitivity. I will have to investigate and fix that in a different PR. I just haven't played around that much with fermions yet.

I put back in `nothing` to the test, since I accidentally left that commented out as I was trying around.